packetbeat/docs/gettingstarted.asciidoc

```diff
@@ -289,7 +297,7 @@ docker run {dockerimage}
 [source,shell]
 ----------------------------------------------------------------------
 sudo chown root packetbeat.yml <1>
-sudo ./packetbeat -e -c packetbeat.yml -d "publish"
+sudo ./packetbeat run -e -c packetbeat.yml -d "publish"
```

Comment:
@monicasarbu Right, but I figured we added run for a reason and might want to show it. Do you think it's better to show the command without including `run`?